spec/features/residents_can_see_confirmation_of_repair_request_spec.rb

```diff
@@ -149,7 +149,7 @@
     expect(fake_api).to have_received(:post).with(
       'repairs',
       priority: 'N',
-      problem: "My sink is blocked\n\nRoom: Kitchen",
+      problemDescription: "My sink is blocked\n\nRoom: Kitchen",
```

Comment:
Are we also getting this new key back in the responses? If so should we update our stubs (above) so that they are more realistic? Currently they include the `problem` key.